src/extensions/extension-discovery.ts

```diff
@@ -213,24 +214,31 @@ export class ExtensionDiscovery {
     }
   }
 
-  protected async getByManifest(manifestPath: string, { isBundled = false, isEnabled = isBundled }: {
+  protected async getByManifest(manifestPath: string, { isBundled = false }: {
     isBundled?: boolean;
-    isEnabled?: boolean;
   } = {}): Promise<InstalledExtension | null> {
     let manifestJson: LensExtensionManifest;
+    let isEnabled: boolean;
 
     try {
       // check manifest file for existence
       fs.accessSync(manifestPath, fs.constants.F_OK);
 
       manifestJson = __non_webpack_require__(manifestPath);
+      const installedManifestPath = path.join(this.nodeModulesPath, manifestJson.name, "package.json");
       this.packagesJson.dependencies[manifestJson.name] = path.dirname(manifestPath);
 
+      if (!isBundled) {
```

Comment:
[nit] mutation could be avoided: `const isEnabled = isBundled || extensionsStore.isEnabled(installedManifestPath);`

Reply:
👍 will change